v1/pipeline.json

```diff
@@ -16181,6 +16181,7 @@
               "$ref" : "#/definitions/pipeline/common/StepSpecType"
             }, {
               "type" : "object",
+              "required" : [ "connectorRef", "image" ],
```

Comment:
will this not break existing pipeline?

Reply:
this is v1, it shouldn't impact anything (Its not used anywhere)